flake8_tidy_imports.py

```diff
@@ -137,78 +137,76 @@ def rule_I201(self, node):
                 )
 
     python2to3_banned_modules = {
-        '__builtin__': 'moved in Python 3. six.moves.builtins can be used as a drop-in replacement',
-        '_winreg': 'moved in Python 3. six.moves.winreg can be used as a drop-in replacement',
-        'anydbm': 'moved in Python 3. dbm can be used as a drop-in replacement',
+        '__builtin__': 'use six.moves.builtins',
+        '_winreg': 'use six.moves.winreg',
+        'anydbm': 'use dbm',
         'asynchat.fifo': 'removed in Python 3',
         'audiodev': 'removed in Python 3',
-        'BaseHTTPServer': 'moved in Python 3. six.moves.BaseHTTPServer can be used as a drop-in replacement',
+        'BaseHTTPServer': 'use six.moves.BaseHTTPServer',
         'Bastion': 'removed in Python 3',
-        'bsddb185': 'moved in Python 3. Use bsddb3 instead',
+        'bsddb185': 'use bsddb3',
         'Canvas': 'removed in Python 3',
         'cfmfile': 'removed in Python 3',
-        'CGIHTTPServer': 'moved in Python 3. six.moves.CGIHTTPServer can be used as a drop-in replacement',
+        'CGIHTTPServer': 'use six.moves.CGIHTTPServer',
         'cl': 'removed in Python 3',
-        'commands': 'moved in Python 3. Use subprocess instead',
-        'compiler': 'moved in Python 3. Use ast instead',
-        'ConfigParser': 'moved in Python 3. six.moves.configparser can be used as a drop-in replacement',
-        'contextlib.nested': 'moved in Python 3. Use Use the contextlib2.ExitStack backport or the shim in http://stackoverflow.com/a/39158985/303931 instead',  # noqa:E501
-        'Cookie': 'moved in Python 3. six.moves.http_cookies can be used as a drop-in replacement',
-        'cookielib': 'moved in Python 3. six.moves.http_cookiejar can be used as a drop-in replacement',
-        'copy_reg': 'moved in Python 3. six.moves.copyreg can be used as a drop-in replacement',
-        'cPickle': 'moved in Python 3. six.moves.cPickle can be used as a drop-in replacement',
-        'cStringIO': 'moved in Python 3. io can be used as a drop-in replacement',
-        'cStringIO.cStringIO': 'moved in Python 3. io.BytesIO can be used as a drop-in replacement',
-        'cStringIO.StringIO': 'moved in Python 3. six.moves.cStringIO can be used as a drop-in replacement',
-        'Dialog': 'moved in Python 3. six.moves.tkinter_dialog can be used as a drop-in replacement',
+        'commands': 'removed in Python 3, use subprocess',
```

Comment:
I like most of the shorter messages, since "moved in python 3" doesn't add extra extra information, but I'm a fairly disappointed to see the "drop-in replacement" language removed.

Some libraries are not precise substitutes. "Drop-in replacement" means that I devoted effort to checking they're the same. For example, this PR makes `commands` (for which subprocess is _not_ a drop-in replacement) have the same message as `ConfigParser` (for which `six.moves.configparser` _is_ a drop-in replacement). Some of the other cases may in fact be exact substitutes or I may have made errors, but I've found this to be useful information.

If I'd known you were going to remove those, I would not have bothered to submit the PR originally, since this will make the released version much less useful than my previous bulky configuration file.